src/runtime12.js

```diff
@@ -67,7 +67,7 @@ function XMLHttpRequest() {
 }
 
 XMLHttpRequest.prototype.open = function(method, url, is_async) {
-    this.is_async = is_async
+    this.is_async = is_async;
```

Comment:
```suggestion
    this.is_async = is_async
```

Reply:
Why are you against this semicolon???

Reply:
Oh - this is JS. sorry.

Reply:
I was wondering why you had it out for this one semicolon 😆